extensions/positron-python/python_files/posit/positron/tests/test_positron_ipkernel.py

```diff
@@ -85,6 +85,51 @@ def test_view_unsupported_type(
     assert capsys.readouterr().err == "UsageError: cannot view object of type 'object'\n"
 
 
+def test_view_simple_expression(shell: PositronShell, mock_dataexplorer_service: Mock) -> None:
+    """Test that %view can evaluate a simple expression."""
+    shell.run_cell("x = 5")
+    expected_result = 6
+    shell.run_cell('%view "x + 1"')
+    mock_dataexplorer_service.register_table.assert_called_once()
+    args, kwargs = mock_dataexplorer_service.register_table.call_args
+    assert args[0] is expected_result  # First arg is the object
+    # Check that the title is either the quoted or unquoted expression (platform-dependent)
```

Comment:
Interesting, is that intentional on our side?

Reply:
It seems to be caused by something internal in the ipkernel machinery, don't think it's anything we are doing